packages/binding-http/test/http-server-test.ts

```diff
@@ -1,15 +1,15 @@
 /********************************************************************************
  * Copyright (c) 2018 - 2020 Contributors to the Eclipse Foundation
- * 
+ *
```

Comment:
there are many whitespace diffs. do you think we can remove them again.

Reply:
Whoops, my bad.  I'll remove them.